judgments/templatetags/document_utils.py

```diff
@@ -8,14 +8,14 @@
 
 
 @register.filter
-def get_title_to_display_in_html(document_title, document_noun):
-    if not document_title:
+def get_title_to_display_in_html(document):
```

Comment:
Super picky - could we add type hints for this?

Reply:
Have done that (and added to the function below too)